cmd/stack_lint.go

```diff
@@ -24,6 +24,9 @@ import (
 	"github.com/spf13/cobra"
 )
 
+var errorCount int
```

Comment:
Since we haven't changed the package yet, I suggest that these vars (which are now globally scoped to the `cmd` package -i.e. the entire universe) are called `stackLintErrorCount` and `stackLintWarningCount`.

Reply:
Done :)